### PR TITLE
refactor examples. updates to kc22

### PR DESCRIPTION
**Pull request by xgp**

### CODE REVIEW

File: examples/postgres/docker-compose.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ services:
     ports:
       - 5433:5432
   keycloak:
-    image: phasetwo/phasetwo-keycloak-test-crdb:latest
+    image: quay.io/phasetwo/phasetwo-keycloak:latest
     environment:
       PROXY_ADDRESS_FORWARDING: 'true'
       KEYCLOAK_USER: admin
```
Comment (m-bull) [inline examples/postgres/docker-compose.yml:24]:
```suggestion
      KEYCLOAK_USER: admin
      KC_DB: postgres
```

Or you end up using the default `dev-file` and wondering where all your data goes when you bump your container tag :smile:

Comment (xgp) [reply examples/postgres/docker-compose.yml:24]: LOL. Whoops. Thanks for the PR

**Comment by xgp** [comment]
We are gated by our custom admin UI additions. The Keycloak team goes out of their way to make sure it is massively hard to upgrade any admin UI customizations, despite the "best practice" being to fork and extend it. That said, we're fairly close on porting our additions, so it should be this week we release our 22 image.

**Comment by xgp** [comment]
Well, there were ways to extend the themes for everything before they embarked on using React for everything. They killed that for admin, and now account, and they will probably soon do it to login as well. Remember, Red Hat is the only customer that Keycloak is built for. You and I are irrelevant to them.

**Pull request merged**